modules/c++/math/include/math/Round.h

```diff
@@ -24,6 +24,8 @@
 #define __MATH_ROUND_H__
 
 #include <cmath>
+#include <except/Exception.h>
```

Comment:
Don't need these two includes in header since the implementation is in the cpp now